redis/client.py

```diff
@@ -824,7 +847,7 @@ def __init__(self, host='localhost', port=6379,
                  ssl_check_hostname=False,
                  max_connections=None, single_connection_client=False,
                  health_check_interval=0, client_name=None, username=None,
-                 retry=None):
+                 retry=None, redis_connect_func=None):
```

Comment:
Note to self: This is really getting out of control. Far too many items in init now.